lib/new_relic/agent/method_tracer_helpers.rb

```diff
@@ -11,7 +11,7 @@ module MethodTracerHelpers
       #   function: the name of the Ruby method
       #   namespace: the Ruby class' namespace as a string, ex: 'MyModule::MyClass'
       SOURCE_CODE_INFORMATION_PARAMETERS = %i[filepath lineno function namespace].freeze
-      SOURCE_CODE_INFORMATION_FAILURE_METRIC = "Supportabiltiy/CodeLevelMetrics/Ruby/Failure".freeze
+      SOURCE_CODE_INFORMATION_FAILURE_METRIC = "Supportability/CodeLevelMetrics/Ruby/Failure".freeze
```

Comment:
Scary because I can't find anything else related -- it seemed path-like but there wasn't a path item...

Reply:
It does look a lot like a path! This is a [metric](https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types/#metrics) name.

It's a great find because we have internal tooling that reports our metrics with names that begin with "Supportability" and this one wasn't getting picked up due to the misspelling.